spec/take_away_ui_spec.rb

```diff
@@ -0,0 +1,232 @@
+require 'take_away_ui'
+
+describe TakeAway do
+
+  #Doubling cuisine class
+  let(:dummy_cuisine_klass) {double :dummy_cuisine_klass}
+  subject(:menu) {described_class.new dummy_cuisine_klass, dummy_order_klass, dummy_basket_klass}
```

Comment:
Metrics/LineLength: Line is too long. [97/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)